Update readme with latest info from docs page (#810)
zenml-io/zenml: README.md

+ Automatically track and version data, models, and other artifacts.
+ Track model hyperparameters and metrics with experiment tracking tools.
+ Measure and visualize train-test skew, training-serving skew, and data drift.

## 🍰 ZenBytes
New to MLOps? Get up to speed by visiting the [ZenBytes](https://github.com/zenml-io/zenbytes) repo.

>ZenBytes is a series of short practical MLOps lessons taught using ZenML. 
>It covers many of the [core concepts](https://docs.zenml.io/getting-started/core-concepts) widely used in ZenML and MLOps in general.

## 📜 ZenFiles
Already comfortable with ZenML and wish to elevate your pipeline into production mode? Check out [ZenFiles](https://github.com/zenml-io/zenfiles).

>ZenFiles is a collection of production-grade ML use-cases powered by ZenML. They are fully fleshed out, end-to-end projects that showcase ZenML's capabilities. They can also serve as a template from which to start similar projects.

# 👭 Collaborate with your team

ZenML is built to support teams working together. 
The underlying infrastructure on which your ML workflows run can be shared, as can the data, assets, and artifacts in your workflow. 

> You can quickly share your stack with anyone with ZenML by exporting them with:

Reviewer: The concept of "stack" has not been introduced yet, maybe you can add a short intro.

```suggestion
In ZenML, a Stack represents a set of configurations for your MLOps tools and infrastructure. You can quickly share your ZenML stack with anyone by exporting the stack:
```

# 🏇 Get a guided tour with `zenml go`

For a more in-depth introduction to ZenML, taught through Jupyter
notebooks, install `zenml` via pip as described above and type:

```shell
zenml go
```

This will spin up a Jupyter notebook that showcases the above example and how to extend ZenML.

# 🍰 ZenBytes

ZenBytes is a series of short practical MLOps lessons using ZenML and its various integrations. 
It covers many of the core concepts widely used in ZenML and MLOps in general.
If you're new to ZenML this is a good place to start.

After understanding the example in [Quickstart](#🚅-quickstart), your next port of call is the [fully-fleshed-out quickstart
example](https://github.com/zenml-io/zenml/tree/main/examples/quickstart) and
then check out [the ZenBytes repository](https://github.com/zenml-io/zenbytes)
and notebooks.

# 📜 ZenFiles

ZenFiles are production-grade ML use-cases powered by ZenML. They are fully
fleshed out, end-to-end, projects that showcase ZenML's capabilities. They can
also serve as a template from which to start similar projects.

> 
> The ZenFiles project is fully maintained and can be viewed as a sister
> repository of ZenML. Check it out [here](https://github.com/zenml-io/zenfiles).

Reviewer: Would remove that sentence, IMO it's not quite true, ZenFiles is the least-well-maintained code we have I think 😅 

```suggestion
Check out ZenFiles [here](https://github.com/zenml-io/zenfiles)!
```

  board](https://zenml.io/discussion). You can also request for new features here.

> - Start a thread in our [Slack channel](https://zenml.io/slack-invite).

Reviewer: Maybe also mention "create github issue" here? @strickvl or is that even a recommended way of requesting features? I remember we discussed at some point whether all feature requests should go over the roadmap instead, what was the decision on this again?

"Repobeats analytics image")

# 👩‍👩‍👧‍👦 Meet the Team

![Meet the Team](./docs/book/assets/meet_the_team.jpeg)

Have a question that's too hard to express on our Slack? Is it just too much effort to say everything on a 
long GitHub issue? Or are you just curious about what ZenML has been up to in the past week? Well, register now for the ZenML Office (Half) Hour to get your answers and more!
It's free and open to everyone.

> Every week, part of the ZenML [core team](https://zenml.io/company#CompanyTeam) will pop in for 30 minutes to interact directly with the community. Sometimes we'll be presenting a feature. Other times just take questions and have fun. Join us if you are curious about ZenML, or just want to talk shop about MLOps.

Reviewer: ```suggestion
Every week, part of the ZenML [core team](https://zenml.io/company#CompanyTeam) will pop in for 30 minutes to interact directly with the community. Sometimes we'll be presenting a feature. Other times we just take questions and have fun. Join us if you are curious about ZenML, or just want to talk shop about MLOps.
```

We will host the gathering every Wednesday 8:30AM PT (5:30PM CET). 
Register now through [this link](https://www.eventbrite.com/e/zenml-meet-the-community-tickets-354426688767), 
or subscribe to the [public events calendar](https://calendar.google.com/calendar/u/0/r?cid=Y19iaDJ0Zm44ZzdodXBlbnBzaWplY3UwMmNjZ0Bncm91cC5jYWxlbmRhci5nb29nbGUuY29t) to get notified 
before every community gathering.

# 🆘 Getting Help

The first point of call should be [our Slack group](https://zenml.io/slack-invite/).
Ask your questions about bugs or specific use cases, and someone from the [core team](https://zenml.io/company#CompanyTeam) will respond.

> Or if you prefer, [open an issue](https://github.com/zenml-io/zenml/issues/new/choose) on our GitHub repo.

Reviewer: ```suggestion
Or, if you prefer, [open an issue](https://github.com/zenml-io/zenml/issues/new/choose) on our GitHub repo.
```

| 🗃 **[API Guide]** | Build production ML pipelines using class-based or functional API. |
| 🚀 **[New in v0.11.0]** | New features, bug fixes. |
| 🗳 **[Vote for Features]** | Pick what we work on next! |
| 📓 **[Docs]** | Full documentation for creating your own ZenML pipelines. |
| 📒 **[API Reference]** | The detailed reference for ZenML's API. |
| 🍰 **[ZenBytes]** | A guided and in-depth tutorial on MLOps and ZenML. |
| 🗂️️ **[ZenFiles]** | End-to-end projects using ZenML. |
| ⚽️ **[Examples]** | Learn best through examples where ZenML is used? We've got you covered. |
| 📬 **[Blog]** | Use cases of ZenML and technical deep dives on how we built it. |
| 🔈 **[Podcast]** | Conversations with leaders in ML, released every 2 weeks. |
| 📣 **[Newsletter]** | We build ZenML in public. Subscribe to learn how we work. |
| 💬 **[Join Slack]** | Need help with your specific use case? Say hi on Slack! |
| 🗺 **[Roadmap]** | See where ZenML is working to build new features. |
| 🙋‍♀️ **[Contribute]** | How to contribute to the ZenML project and code base. |

[ZenML 101]: https://docs.zenml.io/
[Core Concepts]: https://docs.zenml.io/getting-started/core-concepts

> [API Guide]: https://docs.zenml.io/v/docs/developer-guide/steps-and-pipelines/functional-vs-class-based-api

Reviewer: I think this one should not be here, it's not such a important concept IMO

"Repobeats analytics image")

# 👩‍👩‍👧‍👦 Meet the Team

![Meet the Team](./docs/book/assets/meet_the_team.jpeg)

Have a question that's too hard to express on our Slack? Is it just too much effort to say everything on a 
long GitHub issue? Or are you just curious what ZenML has been up to in the past week? Well, register now for the ZenML Office 
(Half) Hour to get your answers and more!
It's free and open to everyone.

Every week, part of the ZenML core team will pop in for 30 minutes to interact directly with the community. Sometimes we'll be presenting a feature, other times just taking questions, and having fun. Join us if you are curious about ZenML, or just want to talk shop about MLOps.

We will host the gathering every Wednesday 8:30AM PT (5:30PM CET). 
Register now through [this link](https://www.eventbrite.com/e/zenml-meet-the-community-tickets-354426688767), 
or subscribe to the [public events calendar](https://calendar.google.com/calendar/u/0/r?cid=Y19iaDJ0Zm44ZzdodXBlbnBzaWplY3UwMmNjZ0Bncm91cC5jYWxlbmRhci5nb29nbGUuY29t) to get notified 
before every community gathering.

# 🆘 Where to get help

> First point of call should be [our Slack group](https://zenml.io/slack-invite/).

Reviewer: I think you should also mention creating a GitHub issue as an alternative here
